web_apply_field_style/models/base.py

```diff
@@ -0,0 +1,37 @@
+# © 2023 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import api, exceptions, models
+
+logger = logging.getLogger(__name__)
+
+
+class Base(models.AbstractModel):
+    _inherit = "base"
+
+    @api.model
+    def _get_view(self, view_id=None, view_type="form", **options):
+        arch, view = super()._get_view(view_id, view_type, **options)
+        if view_type == "form":
+            self._update_css_class(arch)
+        return arch, view
+
+    def _update_css_class(self, arch):
+        css = self._get_field_styles()
```

Comment:
need to check the whole dict format in an other method instead of complexify current method ?